message_app/static/js/conversations/main.js

```diff
@@ -0,0 +1,79 @@
+// ===== Conversation =====
+const CONVERSATION_PANEL_HTML = document.querySelector("#conversation-container");
+
+class ConversationModel {
```

Comment:
Why do we need a model for conversation in JS too?

Reply:
Two reasons:
- Models on the backend side may not fully serve the purpose of displaying information (what the frontend does), because we designed them to handle app logics and data storage.
- These models serve as the foundations for future migration to a component-based frontend library (React, Angular, Ember)